-------------------------------------------------------------------------------
Test set: itson.sistemarestaurantepersistencia.implementaciones.ClientesDAOTest
-------------------------------------------------------------------------------
Tests run: 40, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 2.970 s <<< FAILURE! -- in itson.sistemarestaurantepersistencia.implementaciones.ClientesDAOTest
itson.sistemarestaurantepersistencia.implementaciones.ClientesDAOTest.registrarClienteMismoTelefonoGeneraExcepcion -- Time elapsed: 0.065 s <<< FAILURE!
org.opentest4j.AssertionFailedError: Expected itson.sistemarestaurantepersistencia.excepciones.ClienteMismoTelefonoExistenteException to be thrown, but nothing was thrown.
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:152)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:73)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:35)
	at org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:3115)
	at itson.sistemarestaurantepersistencia.implementaciones.ClientesDAOTest.registrarClienteMismoTelefonoGeneraExcepcion(ClientesDAOTest.java:458)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)

itson.sistemarestaurantepersistencia.implementaciones.ClientesDAOTest.registrarClienteMismoCorreGeneraExcepcion -- Time elapsed: 0.040 s <<< FAILURE!
org.opentest4j.AssertionFailedError: Expected itson.sistemarestaurantepersistencia.excepciones.ClienteMismoCorreoExistenteException to be thrown, but nothing was thrown.
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:152)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:73)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:35)
	at org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:3115)
	at itson.sistemarestaurantepersistencia.implementaciones.ClientesDAOTest.registrarClienteMismoCorreGeneraExcepcion(ClientesDAOTest.java:425)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)

